suite-common/wallet-core/src/transactions/transactionsReducer.ts

```diff
@@ -200,16 +202,37 @@ export const selectTransactionFirstTargetAddress = memoizeWithArgs(
     },
 );
 
-export const selectTransactionFirstInputAddress = memoizeWithArgs(
+const selectTransactionTargetAddresses = memoizeWithArgs(
```

Comment:
Please move this selectors + utils to our `@suite-native/transactions` package since this code is meant to be used only mobile app. We always aim to minimalize platform specific code in suite-common

Reply:
moved to suite-native in 8e7e8f721